dashboard/templates/leaderboard.html

```diff
@@ -22,28 +17,47 @@ <h2>Weekly Leaderboard ({{date_from}} - {{date_to}})</h2>
                       <tr>
                         <th class="enumerator">#</th>
                         <th>Node</th>
-                        <th>Sessions</th>
-                        <th>Unique Users</th>
+                        <th>Weekly Availability</th>
                         <th>Data Transferred</th>
-                        <th>Weekly Availability (openvpn)</th>
-                        <th>Weekly Availability (wireguard)</th>
+                        <th>Sessions</th>
+                        <th class="mobile-hide">Unique Users</th>
+                        <th class="mobile-hide">Country</th>
+                        <th class="mobile-hide">Status</th>
                       </tr>
                     </thead>
                     <tbody>
                       {% for row in leaderboard_rows %}
-                      <tr>
-                        <td scope="row" class="enumerator">{{loop.index}}</td>
+                      <tr onclick="window.document.location='/node/{{row.provider_id}}/openvpn'">
+                        <td scope="row" class="enumerator">{{index_start+loop.index}}</td>
                         <td class="td-node-key">{{row.provider_id}}</td>
-                        <td>{{row.sessions_count}}</td>
-                        <td>{{row.unique_users}}</td>
+                        <td>{{row.availability}}</td>
                         <td>{{row.data_transferred}}</td>
-                        <td>{{row.availability_openvpn}}</td>
-                        <td>{{row.availability_wireguard}}</td>
+                        <td>{{row.sessions_count}}</td>
+                        <td class="mobile-hide">{{row.unique_users}}</td>
+                        <td class="mobile-hide">{{row.country}}</td>
+                        <td class="mobile-hide">{{row.service_status}}</td>
```

Comment:
So only first 10 people will get bounty - shouldn't we highlight is somehow for people to know if they should expect money?

Reply:
Why only first 10 people ?

Reply:
Ok, my bad - actually everybody will get somebody.
But TOP 10 nodes will get bigger bounties - this is to encourage nodes to be active, not just to register. So we have to show somehow which nodes will get this award.

Reply:
Ok, discussed with Mindaugas and Andrius - we don't need this yet, since this info is given to bounty participants using e-mails :)